pywbemtools/pywbemcli/_pywbem_server.py

```diff
@@ -551,15 +549,13 @@ def connect(self, log=None, use_pull=None, verbose=None):
           ValueError: Other issues with server-related attributes of this
             object that are considered programming errors.
         """
-        # print("Debug: PywbemServer connect")
+        ctx = click.get_current_context()
+        if ctx.obj:
+            if ctx.obj.verbose:
+                click.echo("Connect to server {}".
+                           format(self._mock_server or self._server))
 
-        # TODO(PR900): Remove the PR900 flag test and else path once done.
-        if PR900_REQUIRE_CORRECT_CONNECTION_STATUS:
-            assert self._wbem_server is None  # Must be disconnected
-        else:
-            # Old code to be removed once done
-            if self._wbem_server is not None:
-                return
+        assert self._wbem_server is None  # Must be disconnected
```

Comment:
I have some second thoughts on this. If in some more complex scenario we would actually close twice, it is not helpful for a user if we raise the assertion error. This is great during development but not so great during actual usage. Maybe we can introduce an env var that indicates we are running in test.